scripts/lib/branch.js

```diff
@@ -19,6 +19,9 @@ async function branchNameForIssue (owner, repository, issueNumber, index) {
   const namePart = i.title
     .toLowerCase()
     .replace(/ /g, '-')
+    .replace(/:/g, '-')
+    .replace(/,/g, '-')
+    .replace(/\//g, '-')
```

Comment:
Although, this one is great, you may achieve the same using a single regexp. Or is there any particular reason you decided to go with multiple `replace` calls instead of a single regexp?

Reply:
Well the only reason I did it this way, is because I don't really like writing/reading regexps. Should I rewrite it?

Reply:
A classic quote in the topic:

> Some people, when confronted with a problem, think "I know, I'll use regular expressions." Now they have two problems.

https://blog.codinghorror.com/regular-expressions-now-you-have-two-problems/

A great tool that can help a lot working with regular expressions is https://regexr.com/

Reply:
To be fair, this was not a "difficult" regex to add (hope I got it right), so now it's replaced.

Reply:
Checked it, it's on point 👍 Thanks for the rewrite!